cypress/integration/participantTable.spec.js

```diff
@@ -15,14 +15,18 @@ describe('Participant Table', () => {
     it(`Correctly renders columns for ${role.name}`, () => {
       cy.kcLogin(role.fixture);
       cy.visit('/participant-view');
+      const featureMultiOrgProspecting =
+        localStorage.getItem('FEATURE_MULTI_ORG_PROSPECTING') !== undefined
```

Comment:
Can this be changed to something like this?
```
const featureMultiOrgProspecting =  localStorage.getItem('FEATURE_MULTI_ORG_PROSPECTING') || false;
```

Reply:
That will be a fallacy.